### PR TITLE
List each command on a separate line in list type

### PR DESCRIPTION
**Pull request by infdahai**
Based on [the comment](https://github.com/apache/kvrocks/issues/1512#issuecomment-1609624027_), ~~we add some preset comments to specific the location for the submitters with working on #1512.~~ we arrange the commands in list.cc

### CODE REVIEW

File: src/commands/cmd_list.cc [hunk 1]
```diff
@@ -556,14 +556,19 @@ REDIS_REGISTER_COMMANDS(
     MakeCmdAttr<CommandLPushX>("lpushx", -3, "write", 1, 1, 1),
     MakeCmdAttr<CommandRPushX>("rpushx", -3, "write", 1, 1, 1), MakeCmdAttr<CommandLPop>("lpop", -2, "write", 1, 1, 1),
```
Comment (torwig) [inline src/commands/cmd_list.cc]: Could you also move `MakeCmdAttr<CommandLPop>("lpop", -2, "write", 1, 1, 1),` to a new line? And others to have a clear list of commands.

Comment (infdahai) [reply src/commands/cmd_list.cc]: https://github.com/infdahai/incubator-kvrocks/blob/aab1cf9784eb9e271408d04c45d991c54b1aa9aa/src/commands/cmd_list.cc#L552-L568

How about this?

Comment (torwig) [reply src/commands/cmd_list.cc]: Yes, it works for me. Personally, I think comments like `// impl the X command here` isn't necessary - it explains nothing and actually is useless.

**Comment by infdahai** [comment]
No changes to these commands, just a change of order to make the format reasonable.

File: src/commands/cmd_list.cc [hunk 2]
```diff
@@ -551,19 +551,22 @@ class CommandLMove : public Commander {
...
-    MakeCmdAttr<CommandLIndex>("lindex", 3, "read-only", 1, 1, 1),
-    MakeCmdAttr<CommandLTrim>("ltrim", 4, "write", 1, 1, 1), MakeCmdAttr<CommandLLen>("llen", 2, "read-only", 1, 1, 1),
-    MakeCmdAttr<CommandLSet>("lset", 4, "write", 1, 1, 1),
-    MakeCmdAttr<CommandRPopLPUSH>("rpoplpush", 3, "write", 1, 2, 1),
-    MakeCmdAttr<CommandLMove>("lmove", 5, "write", 1, 2, 1), )
+REDIS_REGISTER_COMMANDS(MakeCmdAttr<CommandBLPop>("blpop", -3, "write no-script", 1, -2, 1),
+                        MakeCmdAttr<CommandBRPop>("brpop", -3, "write no-script", 1, -2, 1),
+                        MakeCmdAttr<CommandLIndex>("lindex", 3, "read-only", 1, 1, 1),
+                        MakeCmdAttr<CommandLInsert>("linsert", 5, "write", 1, 1, 1),
+                        MakeCmdAttr<CommandLLen>("llen", 2, "read-only", 1, 1, 1),  //
```
Comment (torwig) [inline src/commands/cmd_list.cc]: What is the purpose of the `//` at the end of the line?

Comment (infdahai) [reply src/commands/cmd_list.cc]: You are right. I remove this.

File: src/commands/cmd_list.cc [hunk 3]
```diff
@@ -551,19 +551,22 @@ class CommandLMove : public Commander {
...
-    MakeCmdAttr<CommandLSet>("lset", 4, "write", 1, 1, 1),
-    MakeCmdAttr<CommandRPopLPUSH>("rpoplpush", 3, "write", 1, 2, 1),
-    MakeCmdAttr<CommandLMove>("lmove", 5, "write", 1, 2, 1), )
+REDIS_REGISTER_COMMANDS(MakeCmdAttr<CommandBLPop>("blpop", -3, "write no-script", 1, -2, 1),
+                        MakeCmdAttr<CommandBRPop>("brpop", -3, "write no-script", 1, -2, 1),
+                        MakeCmdAttr<CommandLIndex>("lindex", 3, "read-only", 1, 1, 1),
+                        MakeCmdAttr<CommandLInsert>("linsert", 5, "write", 1, 1, 1),
+                        MakeCmdAttr<CommandLLen>("llen", 2, "read-only", 1, 1, 1),
+                        MakeCmdAttr<CommandLMove>("lmove", 5, "write", 1, 2, 1),
+                        MakeCmdAttr<CommandLPop>("lpop", -2, "write", 1, 1, 1),  //
```
Comment (torwig) [inline src/commands/cmd_list.cc:560]: Still one is present :)

**Review by torwig — CHANGES_REQUESTED** [review]

File: src/commands/cmd_list.cc (hunk 3 above)
Comment (infdahai) [reply src/commands/cmd_list.cc:560]: This one will allow the format to be maintained. If I remove this, the code will merge together. And the commands are arranged in the order of the redis documentation

Comment (torwig) [reply src/commands/cmd_list.cc:560]: @infdahai Thank you for the explanation.

**Review by torwig — APPROVED** [review]

**Review by git-hulk — APPROVED** [review]

**Pull request merged**